src/app/mod.rs

```diff
@@ -359,14 +368,15 @@ impl Surreal {
 	/// // Select a specific record from a table
 	/// const person = await db.select('person:h5wxrf2ewk8xjxosxtyc');
 	/// ```
-	pub async fn select(&self, resource: String) -> Result<JsValue, Error> {
+	#[wasm_bindgen]
```

Comment:
Why is this necessary in this case? The other methods don't have it.

Reply:
It's not, thanks for spotting